Detectors/TRD/simulation/include/TRDSimulation/Detector.h

```diff
@@ -87,13 +87,17 @@ class Detector : public o2::Base::DetImpl<Detector>
   template <typename T>
   void addHit(T x, T y, T z, T time, T energy, int trackId, int detId);
 
+  // Create TR hits
+  void CreateTRhit(int);
+
   std::vector<HitType>* mHits = nullptr; ///!< Collection of TRD hits
 
   float mFoilDensity;
   float mGasNobleFraction;
   float mGasDensity;
 
   bool mTRon;  //  Switch for TR simulation
+  TRsim* mTR;  // TR simulation
```

Comment:
do you need this as pointer?

Reply:
Not needed as pointer. I have changed it now.